index.js

```diff
@@ -8,10 +8,53 @@ var setSize = hasSet && setSizeDescriptor && typeof setSizeDescriptor.get === 'f
 var setForEach = hasSet && Set.prototype.forEach;
 var booleanValueOf = Boolean.prototype.valueOf;
 
+function listFmtDefault(objType, size, parts, details) {
+  var parens = '{}', padType = ' ', padParts = ' ';
+  // size = null: The concept of size doesn't easily apply for this type.
+  // size = false: parts.length could be considered as the size but it's
+  //        customary to not display it for this type.
+
+  switch (objType || null) {
+  // We could save lots of cases and achieve more consistency in formating,
+  // but that would be a breaking change so let's defer that for the next
+  // breaking semver jump.
+  case null:
+    break;
+  case 'Array':
+    parens = '[]';
+    padType = objType = '';
+    break;
+  case 'Object':
+    padType = objType = '';
+    break;
+  case 'Error':
+    objType = '[' + details.obj + ']';  // NB: implicit String() conversion
```

Comment:
Then current master should be fixed independent of this PR, because I took the original expression from [index.js#L79](https://github.com/substack/object-inspect/blob/master/index.js#L79) in order to have least impact on compatibility.